src/traversal/__tests__/printRelayOSSQuery-test.js

```diff
@@ -190,6 +190,34 @@ describe('printRelayOSSQuery', () => {
       });
     });
 
+    it('dedupes arguments', () => {
+      const enumValue = 'WEB';
+      const query = getNode(Relay.QL`
+        query FooQuery {
+          defaultSettings {
+            web: notifications(environment: WEB),
+            foo: notifications(environment: $env)
+          }
+        }
+      `, {
+        env: enumValue,
+      });
+      const alias1 = query.getChildren()[0].getSerializationKey();
+      const alias2 = query.getChildren()[1].getSerializationKey();
```

Comment:
The strictest option here is to verify the exact alias used - let's follow the example from line 375 and use `const alias = generateRQLFieldAlias('notifications.environment(WEB)')`, and then `${alias}: ...` for both fields below to make it clear that the alias should be identical.